aixplain/modules/pipeline/designer/pipeline.py

```diff
@@ -108,7 +109,7 @@ def validate_nodes(self):
 
         if not contains_input or not contains_output or not contains_asset:
             raise ValueError(
-                "Pipeline must contain at least one input, output and asset node"  # noqa
+                "Pipeline must contain at least one input, output and 'Outputable' node"  # noqa
```

Comment:
Explain what Outputable is. The user may not know.

Reply:
resolved, thanks.